client/src/app/pages/reports/reports.tsx

```diff
@@ -31,11 +37,20 @@ import { Landscape } from "./components/landscape";
 import { AdoptionPlan } from "./components/adoption-plan";
 import { IdentifiedRisksTable } from "./components/identified-risks-table";
 import { useFetchApplications } from "@app/queries/applications";
+import { useFetchQuestionnaires } from "@app/queries/questionnaires";
 
 export const Reports: React.FC = () => {
   // i18
   const { t } = useTranslation();
 
+  const [isQuestionnaireSelectOpen, setIsQuestionnaireSelectOpen] =
+    React.useState<boolean>(false);
+
+  const [selectedQuestionnaire, setSelectedQuestionnaire] =
+    React.useState<string>("All questionnaires");
+
+  const { questionnaires } = useFetchQuestionnaires();
```

Comment:
Needs to be driven from taken assessments. Otherwise we won't have any calculated risk fields.

Reply:
So this page shouldn't need to use any questionnaire data at all. It is showing a report based on all assessed questionnaires in the database in the form of "assessments". We should be doing a getAssessments() fetch at the root  of the page and driving the data from that.